saleor/settings.py

```diff
@@ -416,3 +418,11 @@ def get_host():
 CELERY_TASK_SERIALIZER = 'json'
 CELERY_RESULT_SERIALIZER = 'json'
 CELERY_RESULT_BACKEND = 'django-db'
+
+# IMPERSONATE SETTINGS
+IMPERSONATE_URI_EXCLUSIONS = [r'^dashboard/']
+IMPERSONATE_CUSTOM_USER_QUERYSET = 'saleor.userprofile.impersonate.get_impersonatable_users'
+IMPERSONATE_GET_PARAM = 'impersonate'
+IMPERSONATE_REDIRECT_URL = '/?action=' + IMPERSONATE_GET_PARAM
```

Comment:
We don't need `IMPERSONATE_GET_PARAM` and `IMPERSONATE_REDIRECT_URL`. The first one is not a setting from this library that we would want to override. The second one was only used to set the `action` parameter which is not used anymore.

Reply:
Sure, removed.